kong/clustering/services/sync/hooks.lua

```diff
@@ -67,11 +67,8 @@ function _M:notify_all_nodes()
     local res, err = kong.rpc:call(node, "kong.sync.v2.notify_new_version", msg)
     if not res then
       if not err:find("requested capability does not exist", nil, true) then
-        ngx_log(ngx_ERR, "unable to notify new version: ", err)
+        ngx_log(ngx_ERR, "unable to notify ", node, " new version: ", err)
       end
-
-    else
-      ngx_log(ngx_DEBUG, "notified ", node, " ", latest_version)
```

Comment:
I think we still need the log for debugging. It will be hard to diagnose if we log nothing when succeeds

Reply:
See comment: https://github.com/Kong/kong-ee/pull/10460#discussion_r1831761668

Reply:
Let's remove it now, If we do need it in the practice we could add them back.

Reply:
@chronolaw With the level debug it's almost free in runtime. Any specific reason we are removing them?

Reply:
@StarlightIbuki we need to be careful with debug logs. Currently we don't have means of disabling just some debug logs or fine tune what is debug logged. While log messages like these may be beneficial when debugging incremental sync, it makes debugging everything else much harder as logs are just flooded with these.